app/models/order_process.rb

```diff
@@ -3,6 +3,9 @@ class OrderProcess < ApplicationRecord
   acts_as_tenant(:tenant)
   acts_as_taggable_on
 
+  belongs_to :pre, :class_name => 'PortfolioItem'
```

Comment:
Overall the PR looks good, I feel `pre` is the only part I'm a little concerned about, the name just does not seem descriptive enough.

Seems `pre_portfolio_item` would probably be the correct term, with maybe `pre_item` as a possible alternative if we are trying to keep it short.

Thoughts?